metadata_service/api/__init__.py

```diff
@@ -33,11 +33,11 @@ def get_with_kwargs(self, *, id: Optional[str] = None, **kwargs: Optional[Any])
                 actual_id: Union[str, int] = int(id) if id.isdigit() else id
                 object = get_object(id=actual_id, **kwargs)
                 if object is not None:
-                    return self.schema().dump(object).data, HTTPStatus.OK
+                    return self.schema().dump(object), HTTPStatus.OK
```

Comment:
What is the line about?

Reply:
is it because we upgrade the mashmallow lib which the API changes?

Reply:
Yes, as said on the description we updated marshmallow to be in line with the amundsen-common version